docs/configuration.asciidoc

```diff
@@ -180,6 +180,24 @@ It also expects that a period (`.`) is used to separate the integer and the frac
 | 1.0     | Double 
 |============
 
+[float]
+[[config-central-config]]
+==== `CentralConfig` (added[1.1])
+
+If set to `true`, the agent makes periodic requests to the APM Server to fetch the latest configuration.
```

Comment:
We should link to the central docs as well for this feature.
https://www.elastic.co/guide/en/kibana/7.3/agent-configuration.html

```suggestion
If set to `true`, the agent makes periodic requests to the APM Server to fetch the latest {kibana-ref}/agent-configuration.html[APM Agent configuration].
```

Reply:
@bmorelli25 Thank you and done. I actually copied that line [from Java Agent docs](https://www.elastic.co/guide/en/apm/agent/java/current/config-core.html#config-central-config) :) and it doesn't have that link. Also neither do [Go](https://www.elastic.co/guide/en/apm/agent/go/current/configuration.html#config-central-config) and  [Node.js](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html) (or is it *neither ... **or** ...*? ;)) Only [Python](https://www.elastic.co/guide/en/apm/agent/python/current/configuration.html) and [Ruby](https://www.elastic.co/guide/en/apm/agent/ruby/current/configuration.html) Agents docs have the link although in Ruby's case it's named as `APM Agent Configuration via Kibana`. So you might want to do something it... ;)

Reply:
Thank you! Yes, some of the Agents got their docs done before the central documentation was ready. I appreciate the research, I'll create an issue to update them.

And yes, _neither_ is correct 😄